src/core.js

```diff
@@ -23,6 +24,9 @@ if (process.env.REACT_APP_MODE === 'local') {
 }
 
 const assets = [mainAsset, dai, eth];
+if (token) {
+  assets.unshift(token);
```

Comment:
Mind elaborating what this does?

Reply:
Assets is an array of all tokens/coins we will display in the app. If the token variable is set here, then we want to add that to the array so we can display it in the list of tokens.

Reply:
OK, I believe a comment would clear that up nicely. But up to you.

Reply:
Actually let's merge it and comment it later.